package.json

```diff
@@ -12,5 +12,33 @@
   "bugs": {
     "url": "https://github.com/orangeui/orange/issues"
   },
-  "homepage": "https://github.com/orangeui/orange#readme"
+  "homepage": "https://github.com/orangeui/orange#readme",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "webpack-dev-server --watch --mode development",
```

Comment:
@matevzgolavsek i added the same script for running the nuxt docs. When merging, lets rename the docs dev script into `dev-docs` or, keep the scripts for docs inside the `/site` folder.